apps/prairielearn/src/lib/file-store.js

```diff
@@ -117,7 +117,7 @@ module.exports.delete = async (file_id, authn_user_id) => {
  */
 module.exports.getStream = async (file_id) => {
   debug(`getStream(): file_id=${file_id}`);
-  const file = await this.get(file_id, 'stream');
+  const file = await module.exports.get(file_id, 'stream');
```

Comment:
Using `this` in place of `module.exports` feels like a perpetual footgun: https://github.com/swc-project/swc/issues/7476. The sooner we switch to ESM, the better!